repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/system-model/system-model.xsd

```diff
@@ -275,13 +283,153 @@
             </xs:annotation>
         </xs:attribute>
 
-        <xs:attribute name="uri-regex" type="xs:string" use="optional" default=".*">
+        <xs:attribute name="uri-regex" type="xs:string" use="optional">
             <xs:annotation>
                 <xs:documentation>
-                    <html:p>Regex of what uri is allowed access.</html:p>
+                    <html:p>
+                        Deprecated: This attribute is deprecated and will be removed in future major release.
+                        Regex of what uri is allowed access.
+                    </html:p>
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:assert vc:minVersion="1.1"
+                   xpathDefaultNamespace="##targetNamespace"
+                   test="not(@uri-regex and uri)"
+                   xerces:message="Cannot define both a deprecated uri-regex attribute and the new uri element"
+                   saxon:message="Cannot define both a deprecated uri-regex attribute and the new uri element"/>
+    </xs:complexType>
+
+
+    <xs:complexType name="FilterCriterion" abstract="true">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>Defines the parent of all the conditional filter processing elements.</html:p>
+            </xs:documentation>
+        </xs:annotation>
+    </xs:complexType>
+    <xs:complexType name="Method">
```

Comment:
Do we want to support a list of methods or make them do an or and an entry for each?

Reply:
As it is right now, it would need to be `Or`'d if more than one was possible.